charts/em-npa/values.saat.template.yaml

```diff
@@ -0,0 +1,6 @@
+npa-app:
+  environment:
+    IDAM_API_BASE_URI: https://preprod-idamapi.reform.hmcts.net:3511
```

Comment:
wrong value for saat, but do you really need the saat file?
I would just move everything to values.template.yaml